### PR TITLE
SHOT-4000: Prep Alias Python API 3.0.0 Release

### PR DESCRIPTION
**Pull request by staceyoue**
- Remove Python 2 support
- Engine to pop up an error dialog if failed to import the api (e.g. unsupported python version)
- All `import alias_api` import statements must be declared at function level in the engine to avoid import errors without popping up dialog (TODO: move alias_api functionality outside of the engine, JIRA ticket [SHOT-4015](https://jira.autodesk.com/browse/SHOT-4015))

Screenshot of attempting to run Alias with Python 2 from SG Desktop:
api-python2-error

@rob-aitchison let me know if wording for the error message is ok -- do we want to specify an exact Python 3 version, or is >= 3 ok? We compile the API with 3.7 but it will run with at least >= 3

### CODE REVIEW

File: api/alias_api/__init__.py [hunk 1]
```diff
@@ -56,19 +60,16 @@ def import_alias_api():
...
+    alias_api = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(alias_api)
 
     # add the newly created module oject to sys.modules and remap the globals accessor to point at our new module
     sys.modules["alias_api"] = alias_api
     globals()["alias_api"] = sys.modules["alias_api"]
 
 
+if sys.version_info.major < 3:
+    raise Exception("Alias Python API supports Python version >= 3. Using Python version {}".format(sys.version))
```
Comment (rob-aitchison) [inline api/alias_api/__init__.py:73]: "The Alias Python API now only supports Python 3. You are using version {} of Python. Please refer to this [page](https://github.com/shotgunsoftware/tk-alias/wiki/Python-Version-Support) for additional information."

**Review by rob-aitchison — COMMENTED** [review]
I am trying to keep it as "vague" as possible - what do you think of this change?

File: api/alias_api/__init__.py (hunk 1 above)
Comment (staceyoue) [reply api/alias_api/__init__.py]: Updated error message now looks like this:
api-python2-error-edited

**Review by rob-aitchison — APPROVED** [review]
Awesome!

**Pull request merged**